ui-templates/smoldyn_data.json

```diff
@@ -7,7 +7,8 @@
                 "name": "File",
                 "data_type": "file",
                 "description": "A Smoldyn file",
-                "required": true
+                "required": true,
+                "match": ".*\\\\.txt$"
```

Comment:
Does this expression mean that we expect to have "\\" be a part of the path, and if so why? I'm a little confused, but also my regex skills are rusty. Other than that, LGTM!

Reply:
I have to escape the `\` character for JSON by quadrupling it, but [here's](https://regex101.com/r/B7k5Jj/1) a regex tester that explains what everything is doing otherwise.